src/test_l1_dsHdmiIn.c

```diff
@@ -306,19 +312,25 @@ void test_l1_dsHdmiIn_negative_dsHdmiInGetNumberOfInputs(void) {
     uint8_t numInputs;
 
     // Step 1: Call dsHdmiInGetNumberOfInputs() without initializing the HDMI input sub-system
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     UT_ASSERT_EQUAL(dsHdmiInGetNumberOfInputs(&numInputs), dsERR_NOT_INITIALIZED);
+    #endif
 
     // Step 2: Initialize the HDMI input sub-system using dsHdmiInInit()
     UT_ASSERT_EQUAL(dsHdmiInInit(), dsERR_NONE);
 
     // Step 3: Call dsHdmiInGetNumberOfInputs() with NULL as the parameter
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     DS_ASSERT_AUTO_TERM_NUMERICAL(dsHdmiInGetNumberOfInputs(NULL), dsERR_INVALID_PARAM);
```

Comment:
In the else case, we need to handle the expected dsERR_NONE

Reply:
done